Enterprise-CMCS/macpro-mdct-qmr
# mdct-2679 - fix OMS bug

### Description
The tiniest change, but big impact!
`qualifier` was in front `category`, making the structure incorrect when going through validation.
A validation flag (Optional Measure Stratification: [Stratification Category] Error) is firing even when data is properly entered in the field. This validation should only fire if a state selects a stratification category but does not enter data for that category. This is happening for all streamlined rates in the Child and Health Home Core Sets, as well as AMR-AD.


### Related ticket(s)


MDCT-2679

---
### How to test
Go to 2023, AMR-AD or any Child or Health Home core sets. Fill out measure and fill out at least 1 Optional Measure Strat, validate, see that it does not have the following error: `Must fill out at least one NDR set.`


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
